server/services/mappers/personalLearningPlanActionPlanMapper.ts

```diff
@@ -64,16 +65,6 @@ const toPersonalLearningPlanGoal = (
   }
 }
 
-const dateComparator = (left: Date, right: Date, direction: 'ASC' | 'DESC' = 'DESC'): number => {
```

Comment:
This has been moved out into a utility class in it's own right (`/utils/dateComparator.ts`) so that it can be used by the new `CuriousService` class as well as here